### PR TITLE
Use go:embed for thirdparty

### PR DESCRIPTION
**Pull request by cristaloleg**
Simplifies handing of 3rd party files.

`npm install` also changed `package-lock.json` pretty heavily, not sure should I commit this too or not. WDYT ?

### CODE REVIEW

**Review by quasilyte — APPROVED** [review]

File: third_party/d3flamegraph/update.sh [hunk 1]
```diff
@@ -21,36 +21,10 @@ D3FLAMEGRAPH_CSS="d3-flamegraph.css"
 
 cd $(dirname $0)
 
-generate_d3_flame_graph_go() {
+install_and_copy() {
```
Comment (aalexand) [inline third_party/d3flamegraph/update.sh:24]: d3 based flame graph is going away in pprof - see #777.

So I would prefer to avoid any changes to it now.

Comment (cristaloleg) [reply third_party/d3flamegraph/update.sh:24]: okay, so I will revert changes in `third_party/d3flamegraph` but will keep in `third_party/svgpan`. Can you confirm that?

Comment (aalexand) [reply third_party/d3flamegraph/update.sh:24]: Yes, this sounds good.

Comment (cristaloleg) [reply third_party/d3flamegraph/update.sh]: Done

**Review by aalexand — APPROVED** [review]

**Comment by aalexand** [comment]
Will merge once CI passes.

**Pull request merged**